bentoml/_internal/bento/bento.py

```diff
@@ -79,12 +79,12 @@ def _Bento_set_fs(bento: "Bento", _: t.Any, new_fs: "FS") -> "FS":
     return new_fs
 
 
-@attr.define(repr=False)
+@attr.define(repr=False, auto_attribs=False)
```

Comment:
@sauyon `bentoml.get` currently raise an AttributionError and failed to access `_info`, the same happens to `_doc`. setting `auto_attribs=False` solved the issue but I'm not entirely sure why that was happening. seem like a bug in attrs to me.